packages/blockchain-wallet-v4-frontend/src/plugin/internal/background.ts

```diff
@@ -21,6 +22,10 @@ chrome.runtime.onConnect.addListener(function (port: chrome.runtime.Port) {
   //   port.disconnect()
   // }, 10000)
 
+  port.onDisconnect.addListener(function () {
```

Comment:
this listener will be executed only if dapp looses connection with extension background. Since you never connect to extension background from extension itself, this will never work

Reply:
Done